mpp/Makefile.am

```diff
@@ -9,6 +9,7 @@ SUBDIRS = include
 # Include .h and .mod files.
 AM_CPPFLAGS = -I${top_srcdir}/include
 AM_CPPFLAGS += -I${top_srcdir}/mpp/include
+AM_CPPFLAGS += -I${top_builddir}/sort
```

Comment:
Since we are using the sort mod, we need to point to its directory in the include.